cli_reference/developer_cli_odo/debugging-applications-in-odo.adoc

```diff
@@ -6,7 +6,6 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 :FeatureName: Interactive debugging in {odo-title}
```

Comment:
This line needs to be removed as well (since it was input into the TP module that followed).